frontends/web/src/routes/account/add/add.tsx

```diff
@@ -109,29 +100,71 @@ class ManageAccount extends Component<Props, State> {
                 apiPost('account-add', {
                     coinCode,
                     name: accountName,
-                }).then((data: ResponseData) => {
-                    if (data.success) {
-                        this.setState({ step: 2 });
-                        //route('/account/' + data.accountCode);
-                    } else {
-                        if (data.errorCode) {
-                            alertUser(this.props.t(`addAccount.error.${data.errorCode}`));
-                        } else if (data.errorMessage) {
-                            alertUser(this.props.t('unknownError', { errorMessage: data.errorMessage }));
+                })
+                    .then((data: ResponseData) => {
+                        if (data.success) {
+                            this.setState({ step: 'success' });
+                            //route('/account/' + data.accountCode);
+                        } else {
+                            this.showError(data);
                         }
-                    }
-                });
+                    })
+                    .catch(error => this.showError(error));
                 break;
             case 'success':
+                // TODO: route to manage accounts to configure tokens
                 route('/account-summary');
                 return;
         }
     }
 
+    private showError = (error) => {
```

Comment:
I would inline all of this in the apiPost(..).then(...) function above, as it is only used there (after the `.catch(...)` are gone).